VERSION

```diff
@@ -1 +1 @@
-1.1.0
\ No newline at end of file
+1.1.1
```

Comment:
Please revert this back. When a new version is released, we will do the version bump.